### DISCUSSION TITLE
Verwendung von vorkompilierten Tools und Toolchains

### DISCUSSION THREAD

**Discussion by kainers1** [Freetz-NG/freetz-ng#216] [Allgemein] [answered]
Da die Verwendung von vorkompilierten Tools und Toolchains [deaktviert wurde](https://github.com/Freetz-NG/freetz-ng/commit/d6c35401204f899f11478ef316b0a003faef9ab9) bzw. aktuell nur mit REAL_DEVELOPER_ONLY__DLTXS möglich ist, wollte ich fragen ob sie weiter gepflegt und verwendet werden können.

Dauer und Platzbedarf sprechen ja deutlich für vorkompilierte Tools und Toolchains: ~30 min/2,4 GB vs ~3,5h/5,5GB

**Comment by fda77** [comment, marked as answer]
Es gibt keinen Grund diese nicht zu nutzen - ausser man bekommt es nicht hin mit apt-get ein paar Dinge nachzuinstallieren.
Damit kann ich ein komplette minimalst-Image in 2 Minuten erstellen. Da ich öfter mal verschieden Geräte teste ist das sehr hilfreich
Leg dir das fehlende Symbol einfach in config/custom.in an.
Demnächst werden die wieder auswählbar, das Deaktivieren war aber nötig um die Defaults in alten .config zu änder. Eine "choice" kann man sonst leider im nachhinein nicht verändern. Hast ja selbst bemerkt dass es funktioniert ip-phone-forum.de/threads/309620/post-2416199 ^^
Ich denk ich mache dann eine neue "user competence" zwischen EXPTERT und DEVELOPER dazu, APTGURU oder so :)

**Reply by kainers1** [reply]
Meine VM ist leider lahm da dauerts länger, aber die auch recht preisgünstig.

Habs eh in die custum.in eingetragen, wollte nur wissen was die Roadmap ist. Der News-Beitrag lässt sich ja in verschiedene Richtungen deuten.

**Reply by fda77** [reply]
https://github.com/Freetz-NG/freetz-ng/commit/6d34a645303b3870b854747417e853b7aba009ee

**Reply by kainers1** [reply]
👍

**Reply by fda77** [reply]
Vielleicht erstell ich mal irgend wann 64bit binaries.  Die werden halt nicht mehr auf 32bit laufen. Ubuntu und Fedora gibts nicht mehr als 32bit. Abgesehen von der LTS von vor 5 Jahren

**Reply by kainers1** [reply]
Ich denke mal dem Großteils der User wird egal sein, Hauptsache es läuft. Inzwischen haben wir ja herausgefunden, wie die Pakete installiert werden. Daher würde ich nicht zu viel Aufwand investieren.

**Reply by fda77** [reply]
Davon bin ich noch nicht überzeugt. Unter dem Link oben sieht man, dass Gismotro nichts kapiert hat. Und er ist nunmal der Maintainer von Freetz-Linux. Daher kann man davon ausgehen dass darin kein multiarch aktiviert sein wird - und jeder einzeln das Problem haben würde wenn die DLs noch per default aktiviert wären
Da nun nur noch wenige die DLs nutzen, steht 64bit experimenten nichts im Wege :)
